icons.md

```diff
@@ -94,7 +97,8 @@ To download, right click on the icon and click on "Save image", make sure it dow
     <th>.SPL<br>.SWF</th>
     <td colspan=9 class="gray"></td>
     <td colspan=3><img alt="" src="/assets/files/icons/1999/macromedia/swf.ico"></td>
-    <td colspan=4><img alt="" src="/assets/files/icons/2003/macromedia/swf.ico"></td>
+    <td colspan=5><img alt="" src="/assets/files/icons/2003/macromedia/swf.ico"></td>
+	<td><img alt="" src="/assets/files/icons/2007/adobe/swf.ico"><img alt="" src="/assets/files/icons/2007/adobe/swf-alt.ico"></td>
```

Comment:
> Adobe Creative Suite 3, commonly abbreviated to CS3, is the successor to Creative Suite 2 / 2.3. CS3 was **announced on March 27, 2007, and shipped on April 16, 2007**

Please make sure that the cells align with the year columns

Reply:
Besides that the PR looks fine, but you should look into why there's a .SWF file with a color swapped version, by looking at their file associations.

Reply:
> 
> 
> > Adobe Creative Suite 3, commonly abbreviated to CS3, is the successor to Creative Suite 2 / 2.3. CS3 was **announced on March 27, 2007, and shipped on April 16, 2007**
> 
> Please make sure that the cells align with the year columns

![image](https://user-images.githubusercontent.com/45898787/140557731-5a1055a1-2065-462e-b1d7-e386928e9075.png)

> but you should look into why there's a .SWF file with a color swapped version, by looking at their file associations.

the flash cs3 install i had was sitting loose in a random folder, there's no file associtations for it, so i can't confirm

Reply:
![image](https://user-images.githubusercontent.com/45898787/140558189-6be05c07-2e60-48d9-a9a8-754786663a54.png)

i also fixed the flash player misaligned table thing. however i do not have any icons for 2007 flash player as it is not in the files of flash cs3.